Apps/VisionOS/VisionOS/ViewUtils/MarkdownTheme.swift

```diff
@@ -0,0 +1,63 @@
+import MarkdownUI
```

Comment:
Is this a 3rd party dependency? If so, how is it installed? The xcode project diff doesn't look like any dependency was installed? 

What doe the design of this app supposed to look like? Do we have a design created that needs to be replicated?

Reply:
> Is this a 3rd party dependency? If so, how is it installed? The xcode project diff doesn't look like any dependency was installed?

In the very first PR I added it alongside CIO SDK and another dependency called Splash.

> What doe the design of this app supposed to look like? Do we have a design created that needs to be replicated?

We don't have any design for the app, I am pretty much putting it together using out of the box components with minimum styling.

The only non-standard piece is the markdown and code syntax highlighting. The markdown used with the font type/size is likely to need a better designer eyes if we are keen on perfecting the UI.
The call for using markdown is due to the fact that the app intends to contain textual content/docs like that mandates it to be easy to read, update, mix with code blocks, call outs, etc and not having to deal with laying out punch of different types of views. This is on purpose so we can use help from @glosier later to update that content